src/review/views.py

```diff
@@ -1438,8 +1439,9 @@ def review_decision(request, article_id, decision):
     email_content = logic.get_decision_content(request, article, decision, author_review_url)
 
     if article.date_accepted or article.date_declined:
```

Comment:
Could this be on one line?

`if (article.date_accepted or article.date_declined) and decision != 'undecline'` ?